### DISCUSSION TITLE
Question to developers

### DISCUSSION THREAD

**Discussion by upgreydd** [Budibase/budibase#5924] [Help]
Hey guys. I didn't find a better way to reach out. I'd love to see these two merged, can you please check and give me some insights on what to add or fix to approve? I really appreciate any help you can provide.

https://github.com/Budibase/budibase/pull/5787
https://github.com/Budibase/budibase/pull/5793

**Comment by upgreydd** [comment]
@shogunpurple thanks for the explanation about tooltips. 
Can you please help me with how to make an icon menu with tooltips? It's more simple to just add a few icons in table rows instead of links like [Edit] [Delete] [Action] which are taking too much space, especially when action is not one word. I didn't find a way to use existing tooltip which you talk about.

**Reply by upgreydd** [reply]
@shogunpurple https://github.com/Budibase/budibase/issues/5743 there was a request about such an option too. Can you please give me an insight into how to do this to get merged? I really need tooltips for icons.

**Reply by shogunpurple** [reply]
@Upgreydd We aren't going to merge it - it doesn't fit with the current platforms design system and uses `svelte-tooltip`, so that's why I closed the PR.

When I was talking about existing tooltips, I mean we have a full design system already in budibase using adobe spectrum - that both the apps and builder use. If we were to add the tooltips feature, it would have to use the tooltip element that we already use from the `bbui` package.

`packages/bbui/src/Tooltip/Tooltip.svelte`

> Can you please help me with how to make an icon menu with tooltips? 
We don't currently have a tooltip component as part of budibase apps.

Just an FYI too - we have a board of issues for contributors here: https://github.com/Budibase/budibase/projects/22
